### PR TITLE
add basic default /QOpenSys/etc/profile

### LINKED ISSUE #57: /QOpenSys/etc/profile should have basic setup inside chroot
When chroot_setup is run, the `/QOpenSys/etc/profile` within the chroot should have, at a minimum:
```
PATH=/QOpenSys/pkgs/bin:$PATH
export PATH
OBJECT_MODE=64
export OBJECT_MODE
```

### PR DESCRIPTION
**Pull request by ThePrez**
Resolves #57

### CODE REVIEW

**Comment by ThePrez** [comment]
Once this lands, I'd like to get a 2.1.4 version available also. Thanks!

**Comment by abmusse** [comment]
LGTM

One comment is that if we added a `.bashrc` containing the proper PATH, we would not need to run: 

`source /QOpenSys/etc/profile` manually after entering the chroot. 

Although that is assuming the user is using `bash`.

File: chroot_setup.sh [hunk 1]
```diff
@@ -522,6 +522,20 @@ for chroot in ${mylist[@]}
...
+export PATH
+OBJECT_MODE=64
+export OBJECT_MODE
+EOF
+fi
+
 printf "\nTo enter Your Chroot"
 printf "\nRUN: chroot $CHROOT_DIR /QOpenSys/usr/bin/sh\n"
+printf "\nTo set up your PATH to pick up RPM packages once inside your chroot"
+printf "\nRUN: source /QOpenSys/etc/profile"
```
Comment (abmusse) [inline chroot_setup.sh]: @ThePrez 

I tested this out the .profile was created successfully.

Although entering the chroot with  `chroot $CHROOT_DIR /QOpenSys/usr/bin/sh` and running `source /QOpenSys/etc/profile` errored out with:

`/QOpenSys/usr/bin/sh: source:  not found`

Using the dot operator `.  /QOpenSys/etc/profile` worked successfully though.

We may want to recommend using the dot operator here should work with both `ksh` and `bash`

**Comment by abmusse** [comment]
Resolved conflict via web interface

**Comment by abmusse** [comment]
@ThePrez this LGTM, I think we should use squash and merge for this PR making these changes into one commit.

**Pull request merged**